frontend/src/components/Form/SimpleFormInputArea.tsx

```diff
@@ -0,0 +1,30 @@
+import { FieldValues, Path, UseFormRegister } from 'react-hook-form';
+
+interface SimpleFormInputAreaProps<T extends FieldValues>
+  extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {
+  id: Path<T>;
+  labelText: string;
+  register?: UseFormRegister<T>;
+}
+
+export default function SimpleFormAreaInput<T extends FieldValues>({
```

Comment:
Please add some comments.

Reply:
This file was added during a rebase.
Would it be ok for you if we addressed this issue in a separate PR?

Reply:
This file was added during a rebase.
Would it be ok for you if we addressed this issue in a separate PR?

Reply:
I don't know what you mean. The changes in https://github.com/ElektraInitiative/PermaplanT/pull/312/files must be **exactly** and **exclusively** the changes you intended. Please verify if this is the case. If it is not the case, please reopen another PR, and cherry pick only commits you want. Or if everything else fails, create a new branch from master, copy the files and create new commits.

Reply:
I rebased this branch locally onto master and pushed it to the repository.
For some reason Github thinks that the changes that were introduced through the rebase are changes that are part of this branch and thus shows them in the diff.

I don't know what excactly I did wrong, but if you or anyone else knows I'd love to learn.

Reply:
Btw. please also test the #436 and see if it works together with base layer.

Would really love to get this merged soon but with unknown/unwanted changes it is not a good idea to merge it.